packages/comet-uswds/src/components/card/card.tsx

```diff
@@ -36,7 +70,25 @@ export const CardBody: React.FC<PropsWithChildren> = ({ children }: PropsWithChi
 export const CardHeader: React.FC<PropsWithChildren> = ({ children }: PropsWithChildren) => {
```

Comment:
I believe you can remove <PropsWithChildren> 

-- The PropsWithChildren generic in React.FC<PropsWithChildren> is redundant since React.FC already includes children. You can simplify those components by directly using React.FC without PropsWithChildren.--

Reply:
Done